grazer/src/main/java/edu/asu/diging/grazer/core/domain/impl/FileUploadFormImpl.java

```diff
@@ -0,0 +1,28 @@
+package edu.asu.diging.grazer.core.domain.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class FileUploadFormImpl {
```

Comment:
this is all pretty messy, just have one class for the form with all the properties that you want on the form. This class does not have to have an interface and should be in the package with the controllers (in a subpackage). And then have your domain class in the domain package, with interfaces etc. Your controller's job is it to take the information submitted through the form and put it into the domain classes to pass on to the service.